### DISCUSSION TITLE
SHA256 mismatch

### DISCUSSION THREAD

**Discussion by bjorntj** [Homebrew/discussions#3869] [Everyday usage]
I am suddenly getting SHA256 mismatch errors, both when trying to run upgrade and also when trying to install new packages.
Is this a known problem? Anything I should do to fix this or?

`==> Upgrading 3 outdated packages:
dart-lang/dart/dart 2.18.3 -> 2.18.4
six 1.16.0_2 -> 1.16.0_3
sass/sass/sass 1.55.0 -> 1.56.0
==> Downloading https://storage.googleapis.com/dart-archive/channels/stable/release/2.18.4/sdk/dartsdk-macos-arm64-release.zip
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/six/manifests/1.16.0_3
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/six/blobs/sha256:0dee50367c6facbfc8f65e8a82bcd3e08d43da262b1adff6ccf943ef5bfaf313
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:0dee50367c6facbfc8f65e8a82bcd3e08d43da262b1adff6ccf943ef5bfaf313?se=2022-11-04T08%3A05%3A00Z&sig=czSz2v3mJBoUOosuLbvQl93amzg4IWl6vhoHth9TuCc%3D&sp=r&spr=https&sr=b&sv=2019-12-12
######################################################################## 100.0%
==> Downloading https://github.com/sass/dart-sass/archive/1.56.0.tar.gz
==> Downloading from https://codeload.github.com/sass/dart-sass/tar.gz/refs/tags/1.56.0
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 1fddc66a11847255434227ab38edc3a0c1dbc53bfa9c3a7bc3393eee142fafbb
  Actual: 077365a9d5e3068d14e00d4db6f15d6cda66f8052e9e0452e8bf298c8de7e8ab
    File: /Users/bjorntj/Library/Caches/Homebrew/downloads/5ed70be2152ac39ed296c68a7aa7edb2010ddfbf5274f2f039ac7223a6e9740b--dart-sass-1.56.0.tar.gz
To retry an incomplete download, remove the file above.`

**Comment by SMillerDev** [comment]
You'll need to take this up with the two taps that ship that software.

**Reply by bjorntj** [reply]
ok, so this is not homebrew, homebrew just facilitate?

**Reply by SMillerDev** [reply]
Yes, any package name with a `/` in it means it's outside Homebrews's control

**Reply by bjorntj** [reply]
ok, thx.. :)
